examples/connection_tester.cpp

```diff
@@ -991,6 +991,7 @@ int main(int argc, char* argv[])
 			if (ret != buf.size())
 			{
 				fprintf(stderr, "write returned: %d (expected %d)\n", int(ret), int(buf.size()));
+				fclose(f);
```

Comment:
hm. there's no fclose() call in the successful path either, which seems like a problem. Do you think it would be possible to do this? maybe that's overkill

   boost::shared_ptr<FILE, int(*)(FILE*)> f(fopen(torrent_name, "w+"), &fclose);